lib/iex/lib/iex/evaluator.ex

```diff
@@ -82,6 +102,13 @@ defmodule IEx.Evaluator do
     end
   end
 
+  defp find_matched_variables(binding, var_prefix) do
+    for {var_name, _value} <- binding,
```

Comment:
Good to know -- I had no idea!  Is it worth adding test coverage for this since it's a simple mistake to make?  What's an example of an expression that would result in a variable name not being an atom?